packages/flutter_tools/lib/src/build_system/build_system.dart

```diff
@@ -481,41 +480,59 @@ class _BuildInstance {
     final Stopwatch stopwatch = Stopwatch()..start();
     bool passed = true;
     bool skipped = false;
-    try {
-      final bool canSkip = await node.computeChanges(environment, fileCache);
+
+    // The build system should produce a list of aggregate input and output
+    // files for the overall build. The goal is to provide this to a hosting
+    // build system, such as Xcode, to configure logic for when to skip the
+    // rule/phase which contains the flutter build. When looking at the
```

Comment:
... contains the flutter build.

[new paragraph to mitigate wall-of-text]

So that this list contains the right items, when looking...